library/core/tests/clone.rs

```diff
@@ -1,3 +1,7 @@
+use core::clone::CloneToUninit;
+use core::mem::MaybeUninit;
+use core::sync::atomic::{AtomicUsize, Ordering::Relaxed};
```

Comment:
```suggestion
#[cfg(panic = "unwind")]
use core::sync::atomic::{AtomicUsize, Ordering::Relaxed};
```

(or move the `use` inside `test_clone_to_uninit_slice_drops_on_panic()`)


```
  error: unused imports: `AtomicUsize` and `Ordering::Relaxed`
   --> library/core/tests/clone.rs:3:26
    |
  3 | use core::sync::atomic::{AtomicUsize, Ordering::Relaxed};
    |                          ^^^^^^^^^^^  ^^^^^^^^^^^^^^^^^
    |
    = note: `-D unused-imports` implied by `-D warnings`
    = help: to override `-D warnings` add `#[allow(unused_imports)]`
```

Reply:
Fixed, I hope.

Is there a way I could check this locally? I tried `./x test library/core --target=wasm32-unknown-unknown` but it fails on `profiler_builtins` before getting to emitting warnings for `core`.